app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -101,6 +101,10 @@
   end
 
   meta_tags["govuk:static-analytics:strip-postcodes"] = "true" if should_strip_postcode_pii?(content_item_hash, local_assigns)
+
+  stepnavs = links_hash[:part_of_step_navs] || []
+  stepnavs_content = stepnavs.map { |stepnav| stepnav[:content_id] }.join(",")
+  meta_tags["govuk:stepnavs"] = stepnavs_content if stepnavs_content
```

Comment:
I think this needs to finish `if stepnavs_content.present?`.  
At the moment, content with no `part_of_step_navs` links will be defaulted to an empty array which results in an empty string (`""`) from the `map`.  The empty string is truthy which is why the tests are failing.